app/version.go

```diff
@@ -2,6 +2,10 @@ package app
 
 import "github.com/urfave/cli/v2"
 
+const (
+	DefaultVersion = "v0.1.1"
```

Comment:
Why not just set the Version variable with this default?

Reply:
I just tried this--and it doesn't work, looks like the value in the cli always takes precedent. I thought the build arg to take precedent. It is a little more reliable I suppose since it is automated.